api/main_endpoints/routes/URLShortener-Endpoints.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const axios = require('axios');
+const router = express.Router();
+const {
+  checkIfTokenSent,
+  checkIfTokenValid,
+} = require('../util/token-functions');
+const {
+  OK,
+  BAD_REQUEST,
+  UNAUTHORIZED,
+  FORBIDDEN,
+  NOT_FOUND,
+} = require('../../util/constants').STATUS_CODES;
+const membershipState = require('../../util/constants').MEMBERSHIP_STATE;
+
+let URL_SHORTENER_BASE_URL = 'http://localhost:8000';
+
+router.get('/listAll', async (req, res) => {
+  const token = req.query.token;
+  const tokenReq = { body: { token } };
+  console.log(tokenReq);
+  if (!token) {   
+    return res.sendStatus(FORBIDDEN);
+  } else if (!checkIfTokenValid(tokenReq, membershipState.OFFICER)) {
+    return res.sendStatus(UNAUTHORIZED);
+  }
+  try {
+    const response = await axios.get(URL_SHORTENER_BASE_URL + '/list');
+    const data = response.data;
+    res.json(data);
+  } catch (err) {
+    res.status(500).json({ error: 'Failed to list URLs' });
```

Comment:
can we return the same response code as what cleezy returns? just to keep it clean